Resources/Prototypes/Nyanotrasen/Catalog/Fills/Vending/Advertisements/repdrobe.yml

```diff
@@ -0,0 +1,4 @@
+- type: advertisementsPack
+  id: RepDrobeAds
+  advertisements:
+    - advertisement-detdrobe-1
```

Comment:
My bad, I had a feeling it would be. How do I essentially tell it to use the Detective's advertisement spiel, or should I just leave it blank for that?

Reply:
it's already using it 
![image](https://user-images.githubusercontent.com/60792108/184917659-331f6ce1-bf9e-4398-bea2-a8b544641ff6.png)

Reply:
Ah- awesome. No issue there then, I must have missed that part. I'll remove the unused file